apps/google-analytics-4/frontend/src/config.ts

```diff
@@ -8,4 +8,7 @@ function getEnvironmentVariable(environmentVariableName: string) {
 
 export const config = {
   backendApiUrl: getEnvironmentVariable('REACT_APP_BACKEND_API_URL'),
+  version: getEnvironmentVariable('REACT_APP_VERSION'),
+  release: getEnvironmentVariable('REACT_APP_RELEASE'),
```

Comment:
I only comment here to pick your brain and learn. So I see this env variable is set by the scripts in package.json. What are the trade-offs of putting this env variable in the .env file and doing something like this `https://stackoverflow.com/questions/34650527/how-to-use-environment-variables-in-package-json` versus what you have now?

Reply:
i put everything except the git commit hash in the dotenv files – unless I'm missing some knowledge, i don't think you can interpolate arbitrary bash commands within a dotenv file? I would be thrilled to learn I'm wrong and be taught a way as I sunk 20 minutes into try to do exactly that! 😅